tests/helper/charts/rancherbackuprestore.go

```diff
@@ -24,15 +27,30 @@ const (
 	RancherBackupRestoreName          = "rancher-backup"
 	RancherBackupRestoreCRDName       = "rancher-backup-crd"
 	BackupRestoreConfigurationFileKey = "../helper/yamls/inputBackupRestoreConfig.yaml"
+	localStorageClass                 = "../helper/yamls/localStoarageClass.yaml"
+	backupSteveType                   = "resources.cattle.io.backup"
+	restoreSteveType                  = "resources.cattle.io.restore"
 )
 
 var (
 	SecretName = namegen.AppendRandomString("bro-secret")
 )
 
+type BackupOptions struct {
+	Name                       string
+	ResourceSetName            string
+	RetentionCount             int64
+	EncryptionConfigSecretName string
+}
+
 // InstallRancherBackupRestoreChart installs the Rancher backup/restore chart with optional storage configuration.
-func InstallRancherBackupRestoreChart(client *rancher.Client, installOptions *InstallOptions, rancherBackupRestoreOpts *RancherBackupRestoreOpts, withStorage bool) error {
-	// Retrieve the server URL setting from Rancher.
+func InstallRancherBackupRestoreChart(
+	client *rancher.Client,
+	installOptions *InstallOptions,
+	rancherBackupRestoreOpts *RancherBackupRestoreOpts,
+	withStorage bool,
+	storageType string,
+) error {
```

Comment:
it's better if we keep in single line, easy to understand, mostly we using single line, in both golang and python
```suggestion
func InstallRancherBackupRestoreChart(client *rancher.Client, installOptions *InstallOptions, rancherBackupRestoreOpts *RancherBackupRestoreOpts, withStorage bool, storageType string) error {
```